### ISSUE TITLE
New cops not enabled by default should not warn when using --only flag

### ISSUE THREAD

**Issue by tas50** [rubocop/rubocop#8265]
## Expected behavior

When running RuboCop with --only abc/xyz that cop should run even if it's a new cop not enabled by default and you shouldn't receive a warning to enable it in the rubocop.yml.

## Actual behavior

```
rubocop --only Style/RedundantAssignment

The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Layout/EmptyLinesAroundAttributeAccessor (0.83)
 - Layout/SpaceAroundMethodCallOperator (0.82)
 - Lint/DeprecatedOpenSSLConstant (0.84)
 - Lint/MixedRegexpCaptureTypes (0.85)
 - Lint/RaiseException (0.81)
 - Lint/StructNewOverride (0.81)
 - Style/AccessorGrouping (0.87)
 - Style/BisectedAttrAccessor (0.87)
 - Style/ExponentialNotation (0.82)
 - Style/HashEachMethods (0.80)
 - Style/HashTransformKeys (0.80)
 - Style/HashTransformValues (0.80)
 - Style/RedundantAssignment (0.87)
 - Style/RedundantFetchBlock (0.86)
 - Style/RedundantRegexpCharacterClass (0.85)
 - Style/RedundantRegexpEscape (0.85)
 - Style/SlicingWithRange (0.83)
For more information: https://docs.rubocop.org/rubocop/versioning.html
Inspecting 304 files
................................................................................................................................................................................................................................................................................................................
```

## Steps to reproduce the problem

Run a newly introduced cop on the CLI without configuring it in the .rubocop.yml

## RuboCop version

```
$ [bundle exec] rubocop -V
0.87.1 (using Parser 2.7.1.4, rubocop-ast 0.1.0, running on ruby 2.7.1 x86_64-darwin19)
```
